index.html

```diff
@@ -0,0 +1,285 @@
+  </head>
+  <body>
+  <div id='body-wrapper'></div>
+  <div id='nav-title-wrapper'>
+    <nav>
+    <div id='nav-details'>
+      <div id="mobile-menu" class='float-left'>
+        <span class="ion-navicon-round float-left"></span>
+      </div>
+      <div id='nav-title'>My Funny Friend and Me</div>
+    </div>
+    <div id='nav-selection'>
+      <ul>
+        <li class="tab"><a href="/" id="homeLink" class="ion-happy maven ivory-text">HOME</a></li>
+        <li class="tab"><a href="#" id="favoritesLink" class="ion-ios-paw maven ivory-text">FAVORITE PETS</a></li>
+        <li class="tab"><a href="#" id="aboutLink" class="ion-person maven ivory-text">ABOUT US</a></li>
+        <li class="tab"><a href="#" id="resourcesLink" class="ion-link maven ivory-text">RESOURCES</a></li>
+      </ul>
+    </div>
+    </nav>
+  </div>
+    <div id='main-wrapper'>
+    <section id='greeting-page' class='section-wrapper'>
+      <div>
+        <h1 id='greeting-text' class='maven ivory-text'>...Your new pet is just a few clicks away! Click here to start...</h1>
+      </div>
+    </section>
+    <section id="stage-1" class='section-wrapper'>
+        <div id='stage-1-wrapper'>
+          <h1 class='maven ivory-text'>I would like to adopt a...</h1>
+        <div class='choice-container'>
+          <button class="petButton" value='dog' id='dog-btn'>DOG</button>
+          <button class="petButton" value='cat' id='cat-btn'>CAT</button>
+        </div>
+        <!-- section where the random pets will go in desktop view -->
+          <h2 id='randomPetMessage' class='maven ivory-text'>Click me for some pets that might interest you.</h2>
+          <div id='randomResultsWrapper'>
+
+          </div>
+      </div>
+    </section>
+
+
+    <section id="searchSection" class='section-wrapper'>
+      <div id="searchSectionWrapper">
+        <h1 class='maven ivory-text'>and I live in...</h1>
+        <form id="zipFindForm">
+          <input id="zipFind" type="text" name="zip" placeholder="zipcode" required>
+          <input class="find-new-pet-btn" type="button" name="findIt" value="Find me a new friend!">
+        </form>
+      </div>
+    </section>
+    <section id="totalMatches" class='section-wrapper'>
+      <div id="newFriends">
+        <h1 class='maven ivory-text'>We found</h1>
+        <h1 id="numMatches" class='maven ivory-text'></h1>
+        <h1 class='maven ivory-text'>potential new friends for you!</h1>
+        <h1 class='maven ivory-text'>We will need more information now to find you that special pet that you would love!</h1>
+        <button id='search-btn-transition' class='maven ivory-text'>OK</button>
+    </div>
+    </section>
+    <section id="filterResults" class='section-wrapper'>
+      <div id="filterResultsWrapper">
+        <h1 class='maven ivory-text'>I would like a new pet who is...</h1>
+        <p class='maven ivory-text font-large'>Please try to limit your criteria to 3 options for best results!</p>
```

Comment:
I would also encourage this restriction to 3 options be a point of validation; i.e. When 3 options have been selected, one must be deselected before another can be activated.